scripts/download-sentry-cli.sh

```diff
@@ -13,7 +13,7 @@ for plat in $PLATFORMS; do
   fi
   echo "${plat}"
   download_url=https://github.com/$REPO/releases/download/$VERSION/sentry-cli-${plat}${suffix}
-  fn="${TARGETDIR}/${plat}${suffix}"
+  fn="${TARGETDIR}/sentry-cli-${plat}${suffix}"
```

Comment:
If this script is going to download Sentry CLI, shouldn't we add a gitignore for the cli path?
`sentry-unity/package-dev/Editor/sentry-cli/`

Reply:
The idea is to ship the package with sentry-cli inside so users won't have to download and set it up themselves.